go.mod

```diff
@@ -126,3 +126,69 @@ require (
 	k8s.io/code-generator v0.17.3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
It's weird to have two `require` blocks. Will a `go mod tidy` clean this up?

Reply:
No, this is the "New Way" ™️ in [1.17](https://golang.org/doc/go1.17#go-command), specifically:

>  If a module specifies go 1.17 or higher in its go.mod file, its go.mod file now contains an explicit require directive for every module that provides a transitively-imported package. (In previous versions, the go.mod file typically only included explicit requirements for directly-imported packages.)

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.

Reply:
![The Mandalorian](https://c.tenor.com/S2_k8jNhy1QAAAAC/the-mandalorian-this-is-the-way.gif)